### PR TITLE
feat: support create pair with provide liquidity

### PR DESCRIPTION
**Pull request by jhlee-young**

### CODE REVIEW

**Review by psy2848048 — APPROVED** [review]

File: content/en/docs/integration/create-your-own-pair.md [hunk 1]
```diff
@@ -49,7 +49,13 @@ Then, you may execute the contract with the organized JSON above.
 
 ## Provide initial liquidity
 
-**Dezswap** pair contract knows the swap rate by the both of the remained assets on the pool. But if you have just created your own pair but no liquidity provided, The contract cannot calculate the rate and all swap & swap simulation raise fail. So, **Dezswap** UI does not list the pair unless the initial liquidity is provided. So, if you want finalize the listing, you should provide the initial liquidity and it should be done on CLI.
+**Dezswap** pair contract derives the swap rate from the amount of the remained assets on the pool. However, if you have just created your own pair and haven't provided its liquidity yet, the contract won't be able to calculate the rate and all swap simulations and transactions will fail. So, for the pair to work successfully, you should provide the initial liquidity.
+
```
Comment (nowooj) [inline content/en/docs/integration/create-your-own-pair.md:57]: It would be nice to add the contents of the increase_allowance to the factory address.

Comment (jhlee-young) [reply content/en/docs/integration/create-your-own-pair.md:57]: [9ea8990](https://github.com/dezswap/docs/commit/9ea8990338e7654a7e7866831cf43c75542a4e26)

File: content/en/docs/integration/create-your-own-pair.md [hunk 2]
```diff
@@ -38,8 +44,8 @@ The JSON message format is as follows:
 This is a JSON constructor of pair contract.
 
 - A token pair can be either, contract-based token, or XPLA Chain native / IBC token
-  - `asset_infos[x].token.contract_addr`: Contract-basd token *address* is entered here.
-  - `asset_infos[x].native_token.denom`: XPLA Chain native / IBC token *denominator* is entered here.
+  - `assets[x].info.token.contract_addr`: Contract-based token *address* is entered here.
+  - `assets[x].info.native_token.denom`: XPLA Chain native / IBC token *denominator* is entered here.
```
Comment (nowooj) [inline content/en/docs/integration/create-your-own-pair.md]: How about `native token(include IBC token)`

File: content/en/docs/integration/create-your-own-pair.md [hunk 3]
```diff
@@ -38,8 +44,8 @@ The JSON message format is as follows:
 This is a JSON constructor of pair contract.
 
 - A token pair can be either, contract-based token, or XPLA Chain native / IBC token
-  - `asset_infos[x].token.contract_addr`: Contract-basd token *address* is entered here.
-  - `asset_infos[x].native_token.denom`: XPLA Chain native / IBC token *denominator* is entered here.
+  - `assets[x].info.token.contract_addr`: Contract-based token *address* is entered here.
```
Comment (nowooj) [inline content/en/docs/integration/create-your-own-pair.md]: I think `cw20` is more specify than `Contract-based`

File: content/en/docs/integration/create-your-own-pair.md (hunk 2 above)
Comment (jhlee-young) [reply content/en/docs/integration/create-your-own-pair.md]: [689d4c0](https://github.com/dezswap/docs/pull/10/commits/689d4c0d8f2dcbefb538837460524dc97d057807)

File: content/en/docs/integration/create-your-own-pair.md (hunk 3 above)
Comment (jhlee-young) [reply content/en/docs/integration/create-your-own-pair.md]: [689d4c0](https://github.com/dezswap/docs/pull/10/commits/689d4c0d8f2dcbefb538837460524dc97d057807)

**Review by nowooj — APPROVED** [review]

**Review by caramis — APPROVED** [review]
LGTM

**Pull request merged**